### DISCUSSION TITLE
Can't figure out what's wrong with my page

### DISCUSSION THREAD

**Discussion by mollyhormann** [CollectionBuilder/collectionbuilder.github.io#122] [General]
I have uploaded all of my photos and metadata to my website, but they only show up in the download section. What did I do wrong? I know very little about code so I am really lost on the whole website. 

https://mollyhormann.github.io/London-metadata/

**Comment by evanwill** [comment]
@mollyhormann it is all very close! 
The only problem right now is the "format" field in your metadata. Right now you have the format value as `Image/JPEG`. This is case sensitive and needs to be lowercase, like `image/jpeg`. The CB-GH template uses "format" to figure out how to display the item, so when it doesn't match up, the image doesn't display correctly. Once you fix that, I think everything will work.
Good luck!

**Reply by mollyhormann** [reply]
Thank you! Now, I think I messed something up for my about page. It is all weird and has random pictures that aren't mine? I tried to change the writing but I think it made it worse
